diesel/src/pg/connection/mod.rs

```diff
@@ -114,6 +114,30 @@ impl Connection for PgConnection {
     fn get_transaction_depth(&self) -> i32 {
         self.transaction_depth.get()
     }
+
+    fn setup_helper_functions(&self) {
+        self.execute(
+            "CREATE OR REPLACE FUNCTION diesel_set_updated_at() RETURNS trigger AS $$
+            BEGIN
+                IF (
+                    NEW IS DISTINCT FROM OLD AND
+                    NEW.updated_at IS NOT DISTINCT FROM OLD.updated_at
+                ) THEN
+                    NEW.updated_at := current_timestamp;
+                END IF;
+                RETURN NEW;
+            END;
+            $$ LANGUAGE plpgsql;"
+        ).expect("Error creating function `diesel_set_updated_at()`");
+        self.execute(
+            "CREATE OR REPLACE FUNCTION diesel_manage_updated_at(_tbl regclass) RETURNS VOID AS $$
+            BEGIN
+                EXECUTE format('CREATE TRIGGER set_updated_at BEFORE UPDATE ON %s
+                FOR EACH ROW EXECUTE PROCEDURE diesel_set_updated_at()', _tbl);
+            END;
+            $$ LANGUAGE plpgsql;"
```

Comment:
Two requests on this. First, can you put this in a `.sql` file and use `include_str!` here? Second, can we do this in a single query with `batch_execute` or whatever the function is called? The one on `SimpleConnection`